### PR TITLE
Add sortKey to newInternal.

### PR DESCRIPTION
**Pull request by sw-square**
Nothing too fancy. Each PR will be updating another function, and so the first one's just changing `newInternal`.

### CODE REVIEW

File: v3/client_internal.go [hunk 1]
```diff
@@ -58,6 +60,7 @@ type internalClient struct {
 
 	tableName        string
 	partitionKeyName string
+	sortKeyName      string
```
Comment (ucirello) [inline v3/client_internal.go:63]: I think we are going to get back at this later. The way it is being done, `internalClient` will be `ClientWithSortkey`, and `Client` just a subset of `ClientWithSortkey`. I wonder if could reorganize the code in a way that both `Client` and `ClientWithSortkey` are supersets of `internalClient`.

In other words, `internalClient` is just the common denominator of these two clients, each of which with their own peculiarities. 

Perhaps the better name for `internalClient` would be `commonClient`.

`commonClient` would take private interfaces from the supersets. So for instance, instead of having _one_ query builder both for `Client` and `ClientWithKey`, you'd have `Client.keySearchPredicate()` and `ClientWithSortkey.keySearchPredicate()` that would be inserted into `commonClient.getKey(...)`. Internally, `commonClient.getKey()` doesn't know about anything on sortKey or any other client specific behavior.

**Review by ucirello — CHANGES_REQUESTED** [review]

File: v3/client_internal.go (hunk 1 above)
Comment (sw-square) [reply v3/client_internal.go:63]: Hm, that's not a bad idea—that way, if something else pops up, it will also be easy to adjust.

Comment (zellyn) [reply v3/client_internal.go:63]: My personal opinion is that this is trying to generalize unnecessarily: as far as I can tell, there are only ever going to be two variants: with or without `sortkey`. Putting the common function in the base, and passing in functions for the parts that vary has all the downsides of the traditional Template Method Pattern (having to jump back and forth between the template and the implementations), but with the added annoyance of having to pass in functions for every piece that varies. Even logging with/without the partition key name is going to have to have a callback or logging interface. It makes the code very hard to follow.

Since there are only ever going to be two flavors, I'd just handle it as tidily as possible in the base client, and leave the other two as wrappers that make the API clean.
